conda_build/post.py

```diff
@@ -1687,46 +1687,67 @@ def check_menuinst_json(files, prefix) -> None:
         return
 
     print("Validating Menu/*.json files")
+    for json_file in json_files:
+        _check_one_menuinst_json(join(prefix, json_file))
+
+
+@cache
+def _build_validator(url):
```

Comment:
Shall we restrict which domains can be provided so they can only come from a series of pre-approved locations? e.g. `github.com/conda/menuinst`, `schemas.conda.org`, etc. Otherwise users might be able to simply link the equivalent of "allow-everything" schema and bypass these validations. Not that they result in errors, though.